src/webui/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Is this file used by a specific text editor? If yes it should not be committed to code tree.